app/models/supporting_document.rb

```diff
@@ -0,0 +1,8 @@
+class SupportingDocument < ApplicationRecord
+  SUPPORTING_DOCUMENT = 'supporting_document'.freeze
+  belongs_to :documentable, polymorphic: true
+
+  has_one_attached :file
```

Comment:
This isn't actually being used - you can delete this line and the only thing that fails is the test setup in one controller spec, because we're not actually using the Active Storage API.

Reply:
Not against removing this, but started and felt it was out of scope for this PR so have left it for now.